virttest/env_process.py

```diff
@@ -526,7 +527,11 @@ def validate_memory_resource():
         if (params.get("setup_ksm") == "yes" and
                 params.get("ksm_run", "1") == "1"):
             magnification = 1.2
-        free_mem = "%s KB" % memory.freememtotal()
+        # Test whether is MemAvailable already introduced
+        if 'MemAvailable' in genio.read_file('/proc/meminfo'):
+            free_mem = "%s KB" % memory.read_from_meminfo('MemAvailable')
```

Comment:
Oops, just saw https://github.com/avocado-framework/avocado/pull/1877
It's a bit of uncomfortable for me that it open '/proc/meminfo' twice. So how about this:
```
try:
    free_mem = "%s KB" % memory.read_from_meminfo('MemAvailable')
except XXXException:
    free_mem = "%s KB" % memory.read_from_meminfo('MemFree')
```